index.html

```diff
@@ -5,28 +5,32 @@
     <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
+    <script src="index.js"></script>
     <title>My Portfolio</title>
 </head>
 
 <body>
-    <header class="container">
-        <nav class="navbar">
-            <img class="nav-image" src="images/image1.png" alt="image1">
-            <div class="hamburger">
-                <span class="bar"></span>
-                <span class="bar"></span>
-                <span class="bar"></span>
-            </div>
-            <div class="menu_bar">
-                <ul>
-                    <li><a>Hello</a></li>
-                    <li><a>Portfolio</a></li>
-                    <li><a>About</a></li>
-                    <li><a>Contact</a></li>
-                </ul>
-            </div>
+  <!-- Mobile modal menu -->
+    <div id="hello-wrapper">
+        <nav id="menu">
+          <i id="cross" class="fas fa-times cross invisible" onclick="toggleMenu()"></i>
+          <ul>
+            <li><a href="#title_section" rel="noopener noreferrer" onclick="toggleMenu()">Hello</a></li>
+            <li><a href="#project" rel="noopener noreferrer" onclick="toggleMenu()">Portfolio</a></li>
+            <li><a href="#about" rel="noopener noreferrer" onclick="toggleMenu()">About</a></li>
+            <li><a href="#contact" rel="noopener noreferrer" onclick="toggleMenu()">Contact</a></li>
+          </ul>
+          <img class="mos-0" src="images/image_geometry_menu_2.svg" alt="image_geometry_menu_2">
```

Comment:
- [ ] [ OPTIONAL ] You have done fine on creating this nav menu, but please try to fix this broken design as you still need to work o the next submission, check this out 👀 
- The X icon font weight is a bit more on the original design, 
- When the user scrolls your page, the nav menu is not fixed, it scrolls accordingly which is not a good way to present things, please make a few touches on that too.

<img width="664" alt="1" src="https://user-images.githubusercontent.com/77843050/210610908-b313ca15-9936-4089-97e2-5fc47a2acf0d.png">

![5](https://user-images.githubusercontent.com/77843050/210610948-ee163433-9afb-4882-8b89-4e63560d01cd.gif)